### ISSUE TITLE
Android bug?

### ISSUE THREAD

**Issue by icecandy** [ricardoalcocer/acspushmod#20]
This module is used in our project, but I haven't been using it myself so not so familiar with it. I did just notice a dodgy line in the Android options:
Line 84
CloudPush.singleCallback = androidOptions.singleCallback || true;
If androidOptions.singleCallback is false, CloudPush.singleCallback will be assigned the value true which is not the intended value I believe. Have to be careful when checking for the existence of object values that are of type Boolean (same applies to the other options assignments).

**Comment by Claymm** [comment]
Yeah you're right using Underscore awesomeness would be good:

```js
CloudPush.singleCallback = _.has(androidOptions, "singleCallback") ? androidOptions.singleCallback : true;
```
or
```js
_.defaults(androidOptions, { singleCallback: true });
```

**Comment by icecandy** [comment]
Yes I like the _.has method because it "does what it says on the tin" :)

**Comment by clemblanco** [comment]
Have a look at 1.0.4.

**Comment by icecandy** [comment]
Great
